CONTRIBUTION.md

```diff
@@ -110,9 +109,11 @@ $ cmake --build --preset release
 Make your changes and ensure to follow our [Design Decisions](docs/design/README.md).
 
 ### Testing
-Test your changes thoroughly to ensure they do not introduce new issues. This is done by building and running a slang-test from the repository root directory. For more details about slang-test, please refer to a [Documentation on testing](tools/slang-test/README.md).
+Test your changes thoroughly to ensure they do not introduce new issues. This is done by building and running a "slang-test" from the repository root directory. For more details about "slang-test", please refer to a [Documentation on testing](tools/slang-test/README.md).
 
-If you are familiar with Workflow/Actions in github, you can check [Our Workflows](.github/workflows). [Windows-selfhosted.yml](.github/workflows/windows-selfhosted.yml) is a good starting point.
+> Note: One of the options for `slang-test.exe` is `-api`, and it takes an additional keyword to specify which API to test. When the option is `-api all-cpu`, as an example, it means it tests all APIs except CUDA. The minus sign (-) after `all` means "exclude" and you can "include" with plus sign (+) like `-api gl+dx11`.
```

Comment:
...except cpu?

Reply:
Thanks for catching the typo.
I fixed it.